mylar/helpers.py

```diff
@@ -3160,8 +3160,10 @@ def nzb_monitor(queue):
 
             if any([nzstat['status'] == 'file not found', nzstat['status'] == 'double-pp']):
                 logger.warn('Unable to complete post-processing call due to not finding file in the location provided. [%s]' % item)
+            elif nzstat['status'] == 'nzb removed':
+                logger.warn('NZB seems to have been removed from queue: %s' % item['nzo_id'])
             elif nzstat['status'] is False:
-                logger.info('Could not find NZBID %s in the downloader\'s queue. I will requeue this item for post-processing...' % item['NZBID'])
```

Comment:
We have to keep the reference to NZBID, as that's the return from nzbget.py (the nzbget monitor) - it uses NZBID, whereas SAB uses the nzo_id. So unfortunately leaving it as in the PR will break nzbget usage.